platformio.ini

```diff
@@ -100,6 +100,8 @@ build_flags =
   -D DECODE_SAMSUNG=true
   -D DECODE_LG=true
   -DWLED_USE_MY_CONFIG
+  #For POV Display uncomment following
```

Comment:
The changes to `platformio.ini` are not allowed so please revert and add sample to `platformio_override.sample.ini`.

Reply:
done :)